lib/vanity/adapters/abstract_adapter.rb

```diff
@@ -122,23 +131,23 @@ def ab_assigned(experiment, identity)
       # true, add particpant if not already recorded for this experiment. If
       # implicit is false (default), only add conversion is participant
       # previously recorded as participating in this experiment.
-      def ab_add_conversion(experiment, alternative, identity, count = 1, implicit = false)
+      def ab_add_conversion(experiment_id, alternative, identity, options={})
```

Comment:
Makes sense to condense these into a `options` parameter, but we'll probably have to make this change in a backwards compatible manner as well, see https://github.com/assaf/vanity/pull/317 for an example of that.